README.md

```diff
@@ -76,14 +76,14 @@ You are now ready to login to Aras and check out the examples for customizing no
 6. Save the form.
 7. Click **Design > Parts** in the table of contents (TOC).
 8. Open an existing Part with previous versions (generation > 1).
-9. On the Part form, click **View > Revisions**.
+9. On the Part form, click **Navigate > Revisions**.
```

Comment:
Since this project supports 11 and 12, also indicate steps for 11.
```suggestion
9. On the Part form, click **Navigate > Revisions**.
    * If using Innovator 11.0 SPx, click **View > Revisions**.
```